packages/react-scripts/config/jest/fileTransform.js

```diff
@@ -16,7 +16,11 @@ const path = require('path');
 module.exports = {
   process(src, filename) {
     if (filename.match(/\.svg$/)) {
-      return `module.exports = {ReactComponent: () => ('')};`;
+      return `module.exports = {
+        __esModule: true,
+        default: ${JSON.stringify(path.basename(filename))},
+        ReactComponent: () => null
```

Comment:
Why not return the filename here as well? Seems like that would be more useful in e.g. snapshots.